packs/classfeatures/catharsis-emotion.json

```diff
@@ -0,0 +1,192 @@
+                "itemId": "{item|id}",
+                "key": "ItemAlteration",
+                "mode": "override",
+                "property": "description",
+                "value": [
+                    {
+                        "text": "PF2E.SpecificRule.CatharticMage.{item|flags.pf2e.rulesSelections.catharticMageDedication}.Description"
+                    },
+                    {
+                        "text": "PF2E.SpecificRule.CatharticMage.{item|flags.pf2e.rulesSelections.catharticMageDedication}.Trigger",
+                        "title": "PF2E.SpecificRule.CatharticMage.EmotionalState.Trigger"
+                    },
+                    {
+                        "text": "PF2E.SpecificRule.CatharticMage.{item|flags.pf2e.rulesSelections.catharticMageDedication}.Activation",
+                        "title": "PF2E.SpecificRule.CatharticMage.EmotionalState.Activation"
+                    },
+                    {
+                        "text": "PF2E.SpecificRule.CatharticMage.{item|flags.pf2e.rulesSelections.catharticMageDedication}.Fervor",
+                        "title": "PF2E.SpecificRule.CatharticMage.EmotionalState.Fervor"
+                    },
+                    {
+                        "text": "PF2E.SpecificRule.CatharticMage.{item|flags.pf2e.rulesSelections.catharticMageDedication}.Fallout",
+                        "title": "PF2E.SpecificRule.CatharticMage.EmotionalState.Fallout"
+                    },
+                    {
+                        "text": "PF2E.SpecificRule.CatharticMage.{item|flags.pf2e.rulesSelections.catharticMageDedication}.Focus",
+                        "title": "PF2E.SpecificRule.CatharticMage.EmotionalState.Focus"
+                    }
+                ]
+            },
+            {
+                "key": "ActiveEffectLike",
+                "mode": "override",
+                "path": "flags.pf2e.emotionalCatharsis.emotion",
+                "value": "{item|flags.pf2e.rulesSelections.catharticMageDedication}"
+            },
+            {
+                "key": "RollOption",
+                "option": "emotional-catharsis:{item|flags.pf2e.rulesSelections.catharticMageDedication}"
+            },
+            {
+                "domain": "all",
+                "key": "RollOption",
+                "label": "PF2E.SpecificRule.CatharticMage.Love.Toggle",
+                "option": "benefit-love-focus",
+                "predicate": [
+                    "self:effect:emotional-fervor",
+                    "emotional-catharsis:love"
+                ],
+                "toggleable": true
+            },
+            {
+                "key": "FlatModifier",
+                "predicate": [
+                    "self:effect:emotional-fervor",
+                    "emotional-catharsis:love"
+                ],
+                "selector": "will",
+                "slug": "catharsis-emotion-love",
+                "type": "status",
+                "value": 1
+            },
```

Comment:
Set hide if disabled in order to make this thing not explode the roll dialog for people